PROTO_tests/tests/io_test.py

```diff
@@ -1523,8 +1575,6 @@ def test_csv_read_write(self):
 
 
 class TestImportExport:
```

Comment:
do this tests in this class cover the tests from `import_export_test.py`? or do they need to be updated

Reply:
Thanks for pointing this out, I deleted the `import_export_test.py` file.